src/legacy/state/orders/updaters/GpOrdersUpdater.ts

```diff
@@ -262,5 +264,13 @@ export function GpOrdersUpdater(): null {
     }
   }, [account, chainId, tokensAreLoaded, updateOrders])
 
+  useEffect(() => {
+    clearOrderStorage()
+
+    return function () {
+      clearOrderStorage()
```

Comment:
why not returning just the function directly?

Reply:
Didn't work for some reason, I've tried it.